README.md

```diff
@@ -3,7 +3,7 @@ Enzyme
 
 [![Join the chat at https://gitter.im/enzymejs/enzyme](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/enzymejs/enzyme?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-[![npm Version](https://img.shields.io/npm/v/enzyme.svg)](https://www.npmjs.com/package/enzyme) [![License](https://img.shields.io/npm/l/enzyme.svg)](https://github.com/enzymejs/enzyme/blob/master/LICENSE.md) [![Build Status](https://travis-ci.org/enzymejs/enzyme.svg)](https://travis-ci.org/enzymejs/enzyme) [![Coverage Status](https://codecov.io/gh/enzymejs/enzyme/branch/master/graph/badge.svg)](https://codecov.io/gh/enzymejs/enzyme/branch/master)
+[![npm Version](https://img.shields.io/npm/v/enzyme.svg)](https://www.npmjs.com/package/enzyme) [![License](https://img.shields.io/npm/l/enzyme.svg)](https://github.com/enzymejs/enzyme/blob/master/LICENSE.md) [![Build Status](https://travis-ci.org/enzymejs/enzyme.svg)](https://travis-ci.org/enzymejs/enzyme) [![Coverage Status](https://codecov.io/gh/enzymejs/enzyme/branch/master/graph/badge.svg)](https://app.codecov.io/gh/enzymejs/enzyme/branch/master)
```

Comment:
it’s easier to keep using the shorter url

Reply:
> it’s easier to keep using the shorter url

Yes, **but**:

Go to:

https://enzymejs.github.io/enzyme/

and have a look at the badges at the start page:

```
coverage: unknown
```

This error will go away with this change. That's why I changed the URL.

Reply:
why would changing the destination URL have any impact on the image? I'd expect that to have to do with the image URL instead.

Reply:
> why would changing the destination URL have any impact on the image?

You are right. 

https://enzymejs.github.io/enzyme/ wasn't updated for a long time, that's the reason why the badge is not displaying correctly. I reverted the change to the destination URL The proper way to get rid of the broken badge is to redeploy the site on `github.io`.